pos/app/(main)/report/components/timePicker.tsx

```
@@ -0,0 +1,108 @@
+            ? 12
+            : hourNumber + 12
+          : hourNumber === 12
+          ? 0
+          : hourNumber;
+      onChange(`${hour24.toString().padStart(2, "0")}:${minutes.toString().padStart(2, "0")}`);
+    }
+  };
+
+  const handleMinuteChange = (newMinute: string) => {
+    const minuteNumber = Number.parseInt(newMinute);
+    if (!isNaN(minuteNumber)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)